manifests/ssltools/nssdb.pp

```diff
@@ -0,0 +1,35 @@
+# Creates a nssdb
+class certs::ssltools::nssdb inherits certs {
+  Exec { logoutput => 'on_failure' }
+
+  $nss_db_password_file   = "${::certs::nss_db_dir}/nss_db_password-file"
+  $nssdb_files            = ["${::certs::nss_db_dir}/cert8.db", "${::certs::nss_db_dir}/key3.db", "${::certs::nss_db_dir}/secmod.db"]
+
+  file { $::certs::nss_db_dir:
+    ensure => directory,
+    owner  => 'root',
+    group  => $::certs::qpidd_group,
+    mode   => '0755',
+  } ->
+  exec { 'generate-nss-password':
```

Comment:
Yes, and maybe if we add ```group => $group``` the file resource won't change anything at all (but still should be there to ensure it's correct).